boardswarm/src/registry.rs

```diff
@@ -36,6 +36,9 @@ impl Properties {
         self.properties.get(prop).map(String::as_ref)
     }
 
+    /// Tests if the elements match all of the elements of an iterator.
```

Comment:
The `warn` phrasing in your commit message is a bit strange for a pretty standard and common semantic

Reply:
Changed from `warn` to `remind`.